tests/features/edge_connector_data_flow/__init__.py

```diff
@@ -1,136 +0,0 @@
-
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (testing):** Entire test suite removed for edge connector data flow.

Similar to the asset data flow tests, the removal of edge connector data flow tests could lead to untested functionality. Ensure that any new or modified features resulting from the API client removal are adequately tested.